.travis.yml

```diff
@@ -84,6 +84,12 @@ jobs:
       name: 'End to End Tests'
       script:
         - unbuffer node build-system/pr-check/e2e-tests.js
+      env:
+        - CACHE_NAME=E2EJOB
+    - stage: test
+      name: 'ESM Integration Tests'
```

Comment:
nit: Maybe shorten the name to "ESM Tests" so that in future, you can run other kinds of tests too?

Reply:
done.